src/masonite/stubs/api/AuthenticationController.py

```diff
@@ -0,0 +1,16 @@
+from masonite.controllers import Controller
+from masonite.request import Request
+from masonite.response import Response
+from masonite.authentication import Auth
+
+
+class AuthenticationController(Controller):
+    def auth(self, auth: Auth, request: Request, response: Response):
+        user = auth.attempt(request.input("username"), request.input("password"))
+
+        if user:
+            return {"data": user.generate_jwt()}
```

Comment:
here it means that a new JWT token is generated each time we make a request to this controller, replacing the token in user table with the newly generated ?

And what's the lifetime of this token ?

Also should we rename `data` to `token` ?

Reply:
Yes. Once you have a token you would not need to hit this endpoint anymore. 

The lifetime of the token will be inside the JWT token which I have to add inside the generate_gwt() method. Somehow I still need to hook this all up to the normal feature convention we have so I have access to the configuration settings in the same way we do with other features

Reply:
Yep it would be great. because I feel we would like to define some settings to generate JWT.